application/models/user.go

```diff
@@ -254,7 +251,7 @@ func (u *User) FindByAccessToken(accessToken string) error {
 	}
 
 	*u = userAccessToken.User
-	return nil
+	return userAccessToken.Renew()
```

Comment:
It seems like this should be a separate call, partly so we don't get confused if there's an error and partly because I  wouldn't expect it to have much to do with finding the access token.

Reply:
Agreed. I refactored to make this possible.